content/en/docs/components/pipelines/caching-v2.md

```diff
@@ -0,0 +1,32 @@
++++
+title = "Caching v2"
+description = "Getting started with Kubeflow Pipelines caching v2"
+weight = 50
+
++++
+{{% beta-status
+feedbacklink="https://github.com/kubeflow/pipelines/issues" %}}
+
+Starting from [Kubeflow Pipelines SDK v2](https://www.kubeflow.org/docs/components/pipelines/sdk/v2/) and Kubeflow Pipelines 1.7.0, Kubeflow Pipelines supports step caching capabilities in both standalone deployment and GCP hosted deployment.
+
+## Before you start
+
+This guide tells you the basic concepts of Kubeflow Pipelines caching and how to use it. 
+This guide assumes that you already have Kubeflow Pipelines installed or want to use standalone or GCP hosted deployment options in the [Kubeflow Pipelines deployment 
+guide](/docs/components/pipelines/installation/) to deploy Kubeflow Pipelines.
+
+## What is step caching?
+
+Kubeflow Pipelines caching provides step-level output caching. 
+And caching is enabled by default for all tasks of pipelines built with [Kubeflow Pipelines SDK v2](https://www.kubeflow.org/docs/components/pipelines/sdk/v2/) using `kfp.dsl.PipelineExecutionMode.V2_COMPATIBLE` mode.
+When Kubeflow Pipeline runs a pipeline, it checks to see whether 
+an execution exists in Kubeflow Pipeline with the interface of each pipeline task.
+The task's interface is defined as the combination of the pipeline task spec(base image, command, args), the pipeline task's inputs(artifacts and parameters), and the pipeline task's outputs(artifacts and parameters). 
```

Comment:
"the pipeline task spec(base image, command, args), the pipeline task's inputs(artifacts and parameters), and the pipeline task's outputs(artifacts and parameters)" -> "the pipeline task specification (base image, command, args), the pipeline task's inputs (artifacts and parameters), and the pipeline task's outputs (artifacts and parameters)"

Reply:
done